### PR TITLE
Adds a TeaSpoon to the cutlery

### PR DESCRIPTION
**Pull request by jjokah**
Hello!

I extended the repository including a teaspoon. hope you find it useful :)

Bye,
John.

### CODE REVIEW

File: TeaSpoon.md [hunk 1]
```diff
@@ -0,0 +1 @@
+# Add a tea-spoon
```
Comment (InTheShadows23) [inline TeaSpoon.md:1]: Tea- spoon of sugar

**Comment by InTheShadows23** [comment]
Correct?

File: TeaSpoon.md (hunk 1 above)
Comment (chenyumic) [reply TeaSpoon.md:1]: # เพิ่มช้อนชานํ้าตาล

**Comment by jjokah** [comment]
Yes.

**Pull request closed without merging**